ext/date/php_date.stub.php

```diff
@@ -146,6 +146,9 @@ public static function __set_state(array $array);
     /** @return DateTime */
     public static function createFromImmutable(DateTimeImmutable $object);
 
+    /** @return DateTime */
+    public static function createFromInterface(DateTimeInterface $object);
```

Comment:
As these are new methods, you can actually add a proper `: DateTime` type without the usual backwards compatibility concern.